tests/attacks/inference/attribute_inference/test_white_box_lifestyle_decision_tree.py

```diff
@@ -116,9 +116,10 @@ def transform_feature(x):
         art_warning(e)
 
 
-def test_check_params(art_warning, image_dl_estimator_for_attack):
+@pytest.mark.skip_framework("dl_frameworks")
```

Comment:
Similar as above, why do deep learning frameworks get excluded?

Reply:
This attack only works on decision trees (it's a whitebox attack).